javascript.js

```diff
@@ -0,0 +1,195 @@
+  },
+];
+
+const workContainer = document.querySelector('.works');
+let worksHtml = '';
+
+portifolioData.forEach((data) => {
+  worksHtml += `  <div class="work-item lg-screen">
+          <img src="./image/${data.imageName}" class="work-image"/>
+          <article class="work-details">
+            <h2 class="work-title">${data.name}</h2>
+            <ul class="work-tags">
+              <li class="work-tag">CANOPY</li>
+              <li class="work-tag">Back End Dev</li>
+              <li class="work-tag">2015</li>
+            </ul>
+            <p class="work-description">
+              ${data.description1}
+              </p>
+              <div class="work-tech-stack">
+              ${data.technologies
+    .map((tech) => `<span class="tech-tag">${tech}</span>`)
+    .join('')}
+            </div>
+
+            <button class="work-link" type="button" data-work-array='${JSON.stringify(data)}'>
+              <p class="link-button">See project</p>
+            </button>
+          </article>
+        </div>`;
+});
+
+workContainer.innerHTML = worksHtml;
+
+const Menu = document.getElementById('menus');
+const closeMenu = document.querySelector('.close-btn');
+const openMenu = document.querySelector('.burger');
+
+const showNav = () => {
+  Menu.style.display = 'block';
+};
+
+const hideNav = () => {
+  Menu.style.display = 'none';
+};
+
+closeMenu.addEventListener('click', hideNav);
+openMenu.addEventListener('click', showNav);
+
+// Popup task
+
+const seeProjectButtons = document.querySelectorAll('[data-work-array]');
+if (seeProjectButtons) {
+  const modal = document.querySelector('.modal');
+  seeProjectButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      let modalHtml = '';
+      const workArray = JSON.parse(button.getAttribute('data-work-array'));
+      modalHtml = ` <div class="modal-container">
+          <div class="head">
+            <div class="left">
+              <h2>${workArray.name}</h2>
+              <ul>
+                <li>CANOPY</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+            </div>
+            <button class="closePopup">
+              <img src="./image/Disabled.png" alt="">
+            </button>
+          </div>
+          <div class="contentimg">
+            <img src="./image/${workArray.imageName}" alt="">
+          </div>
+          <div class="details">
+            <p>
+              <span>
+                ${workArray.description1}
+              </span>
+              <span class="moreDetails">
+                ${workArray.description2}
+              </span>
+            </p>
+            <div class="category">
+              <div class="tag">
+                <span>HTML</span>
+                <span>CSS</span>
+                <span>Javascript</span>
+                <span>Ruby</span>
+                <span>Bootstrap</span>
+              </div>
```

Comment:
Great work so far 👍🏼 , I want just to mention two issues with this part : 

- [ ] it should be dynamically generated from technologies attribute of project object 
- [ ] the background color is a bit darker than it appears in figma

kindly make sure to apply this changes otherwise evry thing looks good 👍🏼 

![Capture d’écran du 2023-08-04 21-30-53](https://github.com/VivekRaaZ/popup-window/assets/59707859/09610794-492d-4ac1-b7b7-bf862f6685fb)

**figma**
![Capture d’écran du 2023-08-04 21-30-46](https://github.com/VivekRaaZ/popup-window/assets/59707859/85aabf31-0c5a-4806-9ce3-6658142ac7ed)